reactComponents/src/searchbar/tokenSearch/ResultDetail.tsx

```diff
@@ -223,7 +226,7 @@ export const ResultDetail: FC<DetailType> = (props: DetailType) => {
           active
         />
         <span className="capitalize">
-          {humanizeExchange(selectedPair.exchange)}
+          {exchange?.title && humanizeExchange(exchange.title)}
```

Comment:
do we need to change any documentation or types, or is title already included?

Reply:
this is already included which is great! the exchange.title equates  to the name property in rome frontend widget config